cmd/jsonnet.cpp

```diff
@@ -455,44 +456,53 @@ static bool process_args(int argc,
         return false;
     }
 
-    std::string filename = remaining_args[0];
-    if (remaining_args.size() > 1) {
-        std::cerr << "ERROR: Already specified " << want
-                  << " as \"" << filename << "\"\n"
-                  << std::endl;
-        usage(std::cerr);
+    bool multiple_files_allowed = config->cmd == FMT &&
+                                  (config->fmtTest || config->fmtInPlace);
+    if (!multiple_files_allowed) {
+        std::string filename = remaining_args[0];
+        if (remaining_args.size() > 1) {
+            std::cerr << "ERROR: Already specified " << want
+                      << " as \"" << filename << "\"\n"
+                      << std::endl;
+            usage(std::cerr);
+            return false;
+        }
+    }
+    config->inputFiles = remaining_args;
+    return true;
+}
+
+static bool read_input_contents(std::string filename, std::string *input) {
+    std::ifstream f;
+    f.open(filename);
+    if (!f.good()) {
+        std::string msg = "Opening input file: " + filename;
```

Comment:
Well, it was like that before, I have no idea. Probably b/c that was shorter than using strerror and then cerr. @sparkprime?